dispatcher/frontend-ui/src/components/UpdateUser.vue

```diff
@@ -93,6 +105,26 @@
             parent.toggleLoader(false);
           });
       },
+      addKey() { // request token on API using credentials
+        let parent = this;
+
+        parent.working = true;
```

Comment:
Why not using similar code as for updating user? Using a parent.error seems unnecessary